articles/backup/backup-azure-mars-troubleshoot.md

```diff
@@ -36,7 +36,7 @@ We recommend that you check the following before you start troubleshooting Micro
 
 | Cause | Recommended actions |
 | ---     | ---    |
-| **Vault credentials aren't valid** <br/> <br/> Vault credential files might be corrupt or might have expired. (For example, they might have been downloaded more than 48 hours before the time of registration.)| [Download new credentials](backup-azure-file-folder-backup-faq.md#where-can-i-download-the-vault-credentials-file) from the Recovery Services vault on the Azure portal. Then take these steps, as appropriate: <ul><li> If you've already installed and registered MARS, open the Microsoft Azure Backup Agent MMC console. Then select **Register Server** in the **Actions** pane to complete the registration with the new credentials. <br/> <li> If the new installation fails, try reinstalling with the new credentials.</ul> **Note**: If multiple vault credential files have been downloaded, only the latest file is valid for the next 48 hours. We recommend that you download a new vault credential file.
+| **Vault credentials aren't valid** <br/> <br/> Vault credential files might be corrupt, might have expired or has different file extension than <i>.vaultCredentials</i>. (For example, they might have been downloaded more than 48 hours before the time of registration.)| [Download new credentials](backup-azure-file-folder-backup-faq.md#where-can-i-download-the-vault-credentials-file) from the Recovery Services vault on the Azure portal. Then take these steps, as appropriate: <ul><li> If you've already installed and registered MARS, open the Microsoft Azure Backup Agent MMC console. Then select **Register Server** in the **Actions** pane to complete the registration with the new credentials. <br/> <li> If the new installation fails, try reinstalling with the new credentials.</ul> **Note**: If multiple vault credential files have been downloaded, only the latest file is valid for the next 48 hours. We recommend that you download a new vault credential file.
```

Comment:
The text should be:

or have a different file extension than *.vaultCredentials*.